### DISCUSSION TITLE
v0.4.9

### DISCUSSION THREAD

**Discussion by lbbrhzn** [lbbrhzn/ocpp#324] [Announcements]
## Changes

* prevent exception at close (#322) @lbbrhzn
* increase test converage (#323) @lbbrhzn
* Measure connection latency (#317) @lbbrhzn

## :package: Dependencies

* build(deps): bump release-drafter/release-drafter from 5.17.5 to 5.17.6 (#316) @dependabot
* build(deps): bump pre-commit from 2.16.0 to 2.17.0 in /.github/workflows (#315) @dependabot

## What's Changed
* build(deps): bump pre-commit from 2.16.0 to 2.17.0 in /.github/workflows by @dependabot in https://github.com/lbbrhzn/ocpp/pull/315
* build(deps): bump release-drafter/release-drafter from 5.17.5 to 5.17.6 by @dependabot in https://github.com/lbbrhzn/ocpp/pull/316
* Measure connection latency by @lbbrhzn in https://github.com/lbbrhzn/ocpp/pull/317
* increase test converage by @lbbrhzn in https://github.com/lbbrhzn/ocpp/pull/323
* prevent exception at close by @lbbrhzn in https://github.com/lbbrhzn/ocpp/pull/322

**Full Changelog**: https://github.com/lbbrhzn/ocpp/compare/v0.4.8...v0.4.9

<hr /><em>This discussion was created from the release <a href='https://github.com/lbbrhzn/ocpp/releases/tag/v0.4.9'>v0.4.9</a>.</em>

**Comment by GrantKeymer** [comment]
Unfortunately, the Maximum_Current slider still shows as Unavailable for me.  I used the 'Update' button in HACS to install v0.4.9.  I've also used all the 'Fix Issue' links under Developer Tools, but none of them seemed to help with this issue.  Is there anything else I need to do in order for this slider to work?

**Reply by drc38** [reply]
I had the slider go unavailable for some unknown reason, I simply set the State in Developer tools to 20 and it returned to available. I am still on 4.8 but there should be no reason it doesn't work on all versions.

**Reply by GrantKeymer** [reply]
I've tried this when charging and not charging and the result is the same...

1)  Set Maximum_Current to 16 in Developer Tools
2)  Go back to Overview
3)  The slider is available and says 16
4)  A few seconds later it returns to Unavailable

I can't see anything specifically in the log about this, but some of the entries are truncated and I may be missing the vital one.

Does that give any clue as to what is going on?

**Reply by drc38** [reply]
If you set State to available in Dev tools does that make a difference, or does it go back to unavailable?

**Reply by GrantKeymer** [reply]
I set the state of Maximum_Current to 'available' in Developer Tools.
Went back to Overview and it was showing as available, with the value of 6 Amps that I'd previously set using the Wallbox app.
However, after a few seconds it went back to unavailable again.

**Reply by drc38** [reply]
Hard to know what's going on, I don't have an issue using the HA docker environment

**Reply by GrantKeymer** [reply]
Looking at the logs, I can see that Meter Values are sent every 15 minutes when not charging, so that can't be the cause.

My HassOS uses Docker as part of the image, so there doesn't seem to be much difference between my setup and yours.  The plot thickens 😕
